src/FakeItEasy/ExceptionMessages.resx

```diff
@@ -192,17 +192,6 @@ The following constructors failed:
   <data name="FakeCreationExceptionDefaultMessage" xml:space="preserve">
     <value>Unable to create fake object.</value>
   </data>
-  <data name="FailedToGenerateProxyPattern" xml:space="preserve">
-    <value>FakeItEasy failed to create fake object of type "{0}".
-
-1. The type is not registered in the current IFakeObjectContainer.
-2. The current IProxyGenerator failed to generate a proxy for the following reason:
-
-{1}</value>
-  </data>
-  <data name="UnableToCreateDummyPattern" xml:space="preserve">
-    <value>FakeItEasy was unable to create dummy of type "{0}", register it in the current IFakeObjectContainer to enable this.</value>

```

Comment:
I do agree that I could split this change out into a separate commit. I am not sure about the statement that the change is not related to the PR. In the PR the goal is to remove all the references to IFakeObjectContainer and since this resource entry contains this reference I wanted to remove it. What do you think? Should I put it in a separate commit or do you really want to open another issue/PR?